### PR TITLE
[Paictl] SSH tool for admin to maintain cluster.

### PR DESCRIPTION
**Pull request by ydye**

### CODE REVIEW

File: paictl.py [hunk 1]
```diff
@@ -502,6 +504,49 @@ def update_external_config(self, args):
         external_conf_update.update_latest_external_configuration()
 
 
+class utility(SubCmd):
```
Comment (xudifsd) [inline paictl.py]: use `Utility`

File: paictl.py [hunk 2]
```diff
@@ -502,6 +504,49 @@ def update_external_config(self, args):
...
+    def register(self, parser):
+        utility_parser = parser.add_subparsers(help="utility for maintaining in a easy way.")
+
+        ssh_parser = SubCmd.add_handler(utility_parser, self.cluster_ssh, "ssh")
+        ssh_parser.add_argument("-p", "--config-path", dest="config_path", required=True, help="path of cluster configuration file")
```
Comment (xudifsd) [inline paictl.py]: Do we need -p? Since the config is in configMap.

Comment (ydye) [reply paictl.py]: In this case, just a tool to help admin execute a command in a cluster level. For example, first copy the az-rdma.sh to the machine which has been selected, and then execute the script on all the machine.

**Comment by ydye** [comment]
> What's the purpose of this PR? Will merge into master?

help admin enable az-rdma in cluster.

**Pull request merged**